jdbc/src/main/java/com/yahoo/ycsb/db/flavors/DefaultDBFlavor.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016 YCSB contributors. All rights reserved.
+ * Copyright (c) 2019 YCSB contributors. All rights reserved.
```

Comment:
This should be 2016, 2019 to reflect work from both periods.

Reply:
Thank you for your review! I've made the changes as requested.